core/modules/system/system.install

```diff
@@ -3391,6 +3411,16 @@ function system_update_1087() {
   }
 }
 
+/**
+ * Creates the {invalidated_cache_tags} table if it does not exist.
+ */
+function system_update_1088() {
```

Comment:
This update hook should probably also set `system_emit_cache_tags`, `use_cache_tags_cache`, and `emit_cache_max_age` in `system.core`. It looks like you set `use_cache_tags_cache` to off by default. Will this be left to builders to enable manually until it is stable?